lib/target/sms/classic/sms_crt0.asm

```diff
@@ -94,14 +94,9 @@ int_RASTER:
 
 ;int_VBL: 
     push    hl 
-    ld      hl, timer 
```

Comment:
I can see what this code is trying to do bit incrementing 'A' doesn't set the carry when it wraps to 0 and 'adc 1' will add 1 or 2 if carry is set. So, keeping it simple, just load a 16-bit value, increment it and save it.